tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/telemetryMetrics/util/CounterMetric.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package org.eclipse.microprofile.fault.tolerance.tck.telemetryMetrics.util;
+
+/**
+ * Allows tests to get the value of a counter and compare it with a baseline.
+ * <p>
+ * Most methods on this class will treat a non-existent counter as having a value of zero to allow implementations to
+ * lazily create metrics.
+ * <p>
+ * Most tests should use {@link MetricGetter} to create instances of this class.
+ */
+public class CounterMetric {
```

Comment:
Please rename the classes in this package as well to include `Telemetry` in the name.